scripts/build.py

```diff
@@ -1436,11 +1440,16 @@ def build_sdk(self):
             platforms = build_private.get_target_platforms()
         else:
             platforms = get_target_platforms()
-            # Since we usually want to use the scripts in this package on a linux machine, we'll unpack
-            # it last, in order to preserve unix line endings in the files
-            if 'x86_64-linux' in platforms:
-                platforms.remove('x86_64-linux')
-                platforms.append('x86_64-linux')
+
+        # For the linux build tools (protoc, dlib_shared etc)
+        if 'x86_64-linux' not in platforms:
+            platforms.append('x86_64-linux')
+
+        # Since we usually want to use the scripts in this package on a linux machine, we'll unpack
+        # it last, in order to preserve unix line endings in the files
+        if 'x86_64-linux' in platforms:
+            platforms.remove('x86_64-linux')
+            platforms.append('x86_64-linux')
```

Comment:
It's good to remember that the different headers aren't always the same.
So, we choose linux as the common platform for headers.
It's still possible to have platform specific versions if needed.